test/run.sh

```diff
@@ -4,7 +4,11 @@ set -o pipefail
 output=$(mktemp)
 trap 'rm -- "$output"' EXIT
 NODE_ENV=test karma start | tee "$output"
-if grep -F -e 'WARN LOG:' -e 'WARN [web-server]:' -C5 "$output"; then
+# Search for: warnings from console.warn(), including Vue warnings; Sass
+# warnings; and warnings from Karma.
+if grep -F -e 'WARN LOG:' -e 'Module Warning' -e 'WARN [web-server]:' -C5 "$output"; then
+	# Reset the text format in case the search results contained formatted text.
+	tput sgr0
```

Comment:
The Sass warning I caused was pretty long and was also bold. `grep` displayed the start of the bold text but not the end, so before I added `tput sgr0`, my terminal was still bold after running `npm run test`. Given that `npm run test` displays formatted text, I think this is a possibility to account for in general (not just for Sass warnings).